### DISCUSSION TITLE
Merge two layers

### DISCUSSION THREAD

**Discussion by isdito** [Zulko/moviepy#1907] [Q&A]
Hello,

1.- it is posible merge a video layer and a image (png)?
2.- If I have a 100ftps clip played back at 25fps, I want to slow it down a few points and the final product save at 25fps. If there is a slowdown I want the final clip to increase in time. I couldn't do it with the library.

clip3 = clip2.fl_time( tiempo ,keep_duration=False)

reggards

**Comment by keikoro** [comment]
Please report this as a proper issue instead of using GitHub's Discussions feature, thank you.
